pkg/security/module/server.go

```diff
@@ -27,8 +28,9 @@ import (
 // EventServer represents a gRPC server in charge of receiving events sent by
 // the runtime security system-probe module and forwards them to Datadog
 type EventServer struct {
+	sync.RWMutex
```

Comment:
We discussed that it might not be that impactful, but we could get rid of this one by using the same array of `map[rules.RuleID]*int64` + swap method.